src/utils/sykepengesoknadUtils.ts

```diff
@@ -1,8 +1,18 @@
 import {
+  Soknadstype,
   SporsmalDTO,
   SykepengesoknadDTO,
 } from "@/data/sykepengesoknad/types/SykepengesoknadDTO";
 
+const texts = {
+  utland: "Søknad om å beholde sykepenger utenfor EØS\n",
```

Comment:
Skal linjeskift være med her? 😊
```suggestion
  utland: "Søknad om å beholde sykepenger utenfor EØS",
```

Reply:
Ja, den lå der fra før da jeg kopierte over fra `SoknadTeaser.tsx`, så jeg turte ikke fjerne den 🤷🏼‍♂️ Men tipper jeg bare kan gjøre det, egentlig.